src/site/_includes/partials/head.njk

```diff
@@ -52,43 +52,11 @@
   <script>{{ inlineScript | minifyJs | cspHash | safe }}</script>
 {% endfor %}
 
-{# Sourced scripts cannot be authorized by a CSP hash individually, so we need
-to use an inline script loader for any sourced scripts. #}
-{# This works because the CSP includes the 'strict-dynamic' keyword, allowing
-the children of an authorized script to run. #}
-{% set scriptLoader %}
-function loadScript(url, type) {
-  const s = document.createElement('script');
-  s.src = url;
-  if (type) {
-    s.type = type;
-  }
-  if (type === 'module') {
-    s.async = false; // Preserve load order.
-    const pre = document.createElement('link');
-    pre.rel = 'modulepreload';
-    pre.href = url;
-    document.head.append(pre);
-    // We use DOMContentLoaded as the loader script is running sync, and inserting a module script here doesn't defer. This brings back normal type="module" behavior.
-    window.addEventListener('DOMContentLoaded', () => {
-      document.head.append(s);
-    });
-  } else {
-    document.head.append(s);
-  }
-}
-loadScript('{{ helpers.hashForProd('/js/app.js') }}', 'module');
-{# Add a facility for pages to declare an array of script paths. #}
-{# If no scripts are declared we will use the default page script. #}
-{# default.js loads the basic set of custom elements that all pages need. #}
-{% for item in pageScripts %}
-  loadScript('{{ helpers.hashForProd(item) }}', 'module');
-{% else %}
-  loadScript('{{ helpers.hashForProd('/js/default.js') }}', 'module');
-{% endfor %}
-{# {% if process.env.ELEVENTY_ENV === 'prod' %} #}
+{# Script loader for sourced scripts, as they cannot be authorized by a CSP hash. #}
+{% include 'partials/script-loader.njk' %}
+{% set apiScripts %}
```

Comment:
How does `apiScripts` end up getting a value? From what I can see, it's just a template variable that will always be `undefined`.

If it's meant to be used elsewhere from, e.g., child templates, could you add some documentation or explanation for it?

Reply:
apiScripts gets value from whatever is between "set" and "endset" tag. The syntax is:

```
{% set variableName %}
  some value here (text)
{% endset %}
```

Reply:
Thanks!